README.md

```diff
@@ -98,6 +98,9 @@ You can also check [http://0.0.0.0:8432](http://0.0.0.0:8432) which should show
 
 ## Run in a Docker container
 
+First, render configs (see above). If you try to run the docker compose before rendering, subsequent rendering may not
+work. Try removing created directories and re-rendering.
```

Comment:
Do you have an example of what the `created directories` might be and how did they get created and why can/should they be deleted?

Reply:
added some more info, ptal